src/sfc/sortijk.f90

```diff
@@ -189,15 +189,15 @@ SUBROUTINE SortElemsByCoords(nElems,ElemBary,nElems_IJK,Elem_IJK)
   CASE(3)
     nElems_IJK(3)=1
     nElems_IJK(1:2)=(/nElems_IJK(2),nElems_IJK(1)/)
-  END SELECT !dir 
-CASE(3)  
+  END SELECT !dir
+CASE(3)
   !nx'=ny*nz
   !ny'=nx*nz
   !nz'=nx*ny
   !nx=sqrt(ny'/nx'*nz') =sqrt(nx*nz*nx*ny/(ny*nz))
   !ny=nz'/nx
   !nz=ny'/nx
-  nElems_IJK(1)=NINT(SQRT(REAL(nElems_IJK(2)*nElems_IJK(3)/nElems_IJK(1))))
+  nElems_IJK(1)=NINT(SQRT(REAL(nElems_IJK(2)*REAL(nElems_IJK(3))/REAL(nElems_IJK(1)))))
```

Comment:
Should the first REAL encompass the whole term or only nElems_IJK(2)? @kopperp

Reply:
You were right, it should only be on `nElems_IJK(2)`. Fixed in ff8f7f3